test/lib/pages/settings_page.rb

```diff
@@ -0,0 +1,43 @@
+class SettingsPage < SitePrism::Page
+  set_url '/settings'
+
+  text_field(:email, 'input#email')
+  label(:email_status, '#email-status')
+  button(:save_email, 'input[value="Update Email"]')
+
+  button(:connect_with_a_different_account, '#trello-connect')
+  label(:trello_connect_status, '#trello-connect-status')
+
+  checkbox(:confirm_delete, '#i-am-sure')
+  button(:choose_to_delete_account, '#delete-account-button')
+  label(:delete_status, 'span#delete-account-status')
+
+  def update_email
+    save_email
+    wait_until { email_status != 'Saving...' }
+  end
+
+  def delete_account
+    choose_to_delete_account
+    wait_until { no_longer_deleting? }
+  end
+
+  alias_method :orig_connect, :connect_with_a_different_account
+  def connect_with_a_different_account(&block)
+    orig_connect
+    block.call(self) if block
```

Comment:
same comment as before in favor of `block_given?`. do a quick search for any other instances.